src/main/scala/nodes/learning/LBFGS.scala

```diff
@@ -183,8 +208,13 @@ class SparseLBFGSwithL2(
     val numCorrections: Int = 10,
     val convergenceTol: Double = 1e-4,
     val numIterations: Int = 100,
-    val regParam: Double = 0.0)
-  extends LabelEstimator[SparseVector[Double], DenseVector[Double], DenseVector[Double]] {
+    val regParam: Double = 0.0,
+    val sparseOverhead: Double = 8)
```

Comment:
It's a constant @etrain and I are using for roughly estimating how much slower an operation on fully filled sparse data is versus the equivalent data stored in dense form. I'll add a comment.